artview/components/nav.py

```diff
@@ -194,54 +194,62 @@ def createHelpUI(self):
     #   Update Methods   #
     ######################
 
-    def _update_InfoUI(self):
-        '''Update the info label.'''
-        self.dirIn = os.path.dirname(self.Vradar.value.filename)
-        self.infodir.setText("Directory: %s"%(self.dirIn))
-        self.infofile.setText("File: %s"%(
-            os.path.basename(self.Vradar.value.filename)))
+#    def _update_InfoUI(self):
+#        '''Update the info label.'''
+#        self.dirIn = os.path.dirname(self.Vradar.value.filename)
+#        self.infodir.setText("Directory: %s"%(self.dirIn))
+#        self.infofile.setText("File: %s"%(
+#            os.path.basename(self.Vradar.value.filename)))
 
     def _update_tools(self):
-        '''Update the info label.'''
-        self._update_InfoUI()
-        filename = self.Vradar.value.filename
-        if filename in self.Vfilelist.value:
-            self.fileindex = self.Vfilelist.value.index(filename)
+        '''Update the navigation button.'''
+        filelist = self.Vfilelist.value
+        if self.filename in filelist:
+            self.fileindex = filelist.index(self.filename)
         else:
             self.fileindex = 0
-        self.act_prev.setToolTip(
-            'Previous file: %s'%os.path.basename(self.Vfilelist.value[self.fileindex - 1]))
-        self.act_next.setToolTip(
-            'Next file: %s'%os.path.basename(self.Vfilelist.value[self.fileindex + 1]))
 
-    #########################
-    #   Selection Methods   #
-    #########################
+        print self.fileindex
+        print filelist
 
-    def get_variables(self):
-        '''Initialize variables.'''
```

Comment:
No, I don't need this at all! All I need is in line 69

        self.filename = ""
        self.fileindex = 0